app/app.js

```diff
@@ -50,6 +50,7 @@ var PatientNew = require('./pages/patientnew');
 var PatientData = require('./pages/patientdata');
 var RequestPasswordReset = require('./pages/passwordreset/request');
 var ConfirmPasswordReset = require('./pages/passwordreset/confirm');
+var SignupVerification = require('./pages/signupverification');
```

Comment:
Part of me doesn't like the name `SignupVerification` and all the variations thereof. I'd prefer `EmailVerification` (since that's actually what we're verifying). But, my feelings aren't super strong on this. Change is part of you agrees, or don't, if you don't :)